.github/workflows/build.yml

```diff
@@ -55,3 +55,35 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           file: ./coverage.info
           format: lcov
+
+  freebsd:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v4
+      - uses: cross-platform-actions/action@e8a7b572196ff79ded1979dc2bb9ee67d1ddb252 # v0.29.0
```

Comment:
Can you replace it with `cross-platform-actions/action@v0.29.0`?

As far as I can remember it came from the curl repository where all the things are pinned. In the avahi repository the actions aren't pinned.